scripts/setup_minikube_kueue.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
```

Comment:
can we add a integration test for this similar to the minikube one for volcano?

You should be able to push to a draft PR (or this one) and test it e2e 

Similar setup to https://github.com/pytorch/torchx/blob/main/.github/workflows/components-integration-tests.yaml#L79-L82